src/main/java/edu/greenblitz/robotName/subsystems/arm/EndEffector/WristUtils/NeoWrist/NeoWrist.java

```diff
@@ -63,6 +61,11 @@ public void moveToAngle(Rotation2d targetAngle) {
         motor.getPIDController().setReference(targetAngle.getRadians(), CANSparkMax.ControlType.kPosition, PID_SLOT, motor.getPIDController().getFF());
```

Comment:
use simple motor feed forward for FF. the getFF of the motorws PIDController is for a constant FF (like only kS)